trusted-firmware-m/CMakeLists.txt

```diff
@@ -11,6 +11,13 @@ set(TFM_VERSION 1.2.0)
 
 ############################ CONFIGURATION #####################################
 
+if (IS_ABSOLUTE "${TFM_PLATFORM}")
```

Comment:
Nothing wrong here, but the commit message states:
> Synchronize with commit a9d5eacdb045a5d793aca0e591ec3abe48fa7205

but I don't find this commit anywhere. 
Could you provide more info ?

Also, are all changes in this commit 319fcf6331fc2e260b0bc28c9f81653ead7a21eb to be reverted once commit a9d5eacdb045a5d793aca0e591ec3abe48fa7205 is merged somewhere ?

Reply:
It's upstream TFM. https://review.trustedfirmware.org/plugins/gitiles/TF-M/trusted-firmware-m/+/a9d5eacdb045a5d793aca0e591ec3abe48fa7205

Reply:
thanks.